Content.Client/PDA/PdaMenu.xaml.cs

```diff
@@ -84,6 +87,17 @@ public PdaMenu()
                 ToHomeScreen();
             };
 
+            CopyStationNameButton.OnPressed += _ =>
+            {
+                _clipboard.SetText(_StationName?? Loc.GetString("comp-pda-ui-unknown"));
```

Comment:
_StationName should be renamed to _stationName, and its not optional so dont need the ?? bit